client/app/bundles/shared/components/Avatar.scss

```diff
@@ -0,0 +1,25 @@
+@import './base';
+
+.avatar {
+  display: inline-block;
+
+  font-family: $font-family;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 22px;
+  text-align: center;
+
+  img {
+    border-radius: 50%;
+    border: 5px solid;
+    border-color: $background-color;
+  }
+
+  div {
+    text-shadow: 0px 2px rgba(0, 0, 0, 0.25);
```

Comment:
Let's tweak to this: `0 2px 4px rgba(0, 0, 0, 0.25)`!

Reply:
@nshki Done!